.github/workflows/builder.yml

```diff
@@ -270,6 +270,18 @@ jobs:
             exit 1
           fi
 
+          echo "Install Core Samba Add-on"
+          test=$(docker exec hassio_cli ha addons install core_samba --no-progress --raw-json | jq -r '.result')
+          if [ "$test" != "ok" ];then
+            exit 1
+          fi
+
+          echo "Start Core Samba Add-on"
```

Comment:
You need to test another add-on or use CLI to adjust the add-on configuration to include password.
<https://github.com/home-assistant/supervisor/runs/3708396503#step:11:227>

Reply:
I reverted it for now. Will later add a command to cli and extend this tests